benchmark/src/main/kotlin/com/google/devtools/ksp/BenchRunner.kt

```diff
@@ -20,16 +20,17 @@ import org.jetbrains.kotlin.cli.jvm.K2JVMCompiler
 import kotlin.system.measureNanoTime
 
 fun main(args: Array<String>) {
-    val runs = 10
-    val warmup = 10
-    val benchName = "Tachiyomi"
+    val runs = 3
```

Comment:
This takes a long time to run, so reducing the number of runs to 3. We might want to parameterize the runner, but we can do that later.